src/decode/lzma2.rs

```diff
@@ -193,3 +190,46 @@ where
 
     Ok(())
 }
+
+#[derive(Debug)]
+/// Raw decoder for LZMA2.
+pub struct Lzma2Decoder {
+    decoder: DecoderState,
```

Comment:
Likewise, in terms of naming this would be better called `lzma_decoder`, to show that the LZMA2 encapsulates a raw LZMA decoder.

Reply:
Since the one in `LzmaDecoder` is `state` how is `lzma_state` to be more consistent?